README.md

```diff
@@ -975,7 +975,6 @@ _Related: [Groupware](#groupware)_
 - [Gossa](https://github.com/pldubouilh/gossa) - Gossa is a light and simple webserver for your files. `MIT` `Go`
 - [IFM](https://github.com/misterunknown/ifm) - Single script file manager. `MIT` `PHP`
 - [miniserve](https://github.com/svenstaro/miniserve) - CLI tool to serve files and dirs over HTTP. `MIT` `Rust`
-- [ResourceSpace](https://www.resourcespace.com) - ResourceSpace open source digital asset management software is the simple, fast, and free way to organise your digital assets. ([Demo](https://www.resourcespace.com/trial), [Source Code](https://www.resourcespace.com/svn)) `BSD-4-Clause` `PHP`
```

Comment:
> https://www.resourcespace.com/versions
> Do It Yourself
>Technically minded and happy configuring a LAMP stack?
> [Subversion / Git Free & 100% Open Source](https://www.resourcespace.com/svn)


```
$ sudo apt install subversion
$ svn co http://svn.resourcespace.com/svn/rs/trunk/
$ cat trunk/license.txt 
Please see the "documentation/licenses" folder for all applicable licenses.
$ ls trunk/documentation/licenses/
chosen.txt          flot.txt          jcrop.txt                    jquery-tag-editor.txt  jquery-ui-touch-punch.txt  lightbox.txt  phpmailer.txt*      tcpdf.txt*   videojs-contrib-hls.txt          wordpress.txt*
ckeditor.md*        font_awesome.txt  jquery-image-annotation.txt  jquery.tshift.txt      jquery-zoom.txt            nuvola.txt*   plupload-2.3.7.txt  tus-php.txt  videojs-resolution-switcher.txt
crystal_icons.txt*  icc.txt           jquery-layout.txt            jquery.txt             jstree.txt                 pdfjs.txt*    resourcespace.txt*  uppy.txt     video.js.txt
live@derrichter:~ $ cat trunk/documentation/licenses/resourcespace.txt
Copyright Oxfam GB, Montala Limited, WWF International, Tom Gleason, David Dwiggins, Historic New England, Colorhythm LLC, Worldcolor, Henrik Frizén 2006-2021
All rights reserved.

Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
	+	Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
	+	Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
	+	Neither the names of any of the copyright holders listed above nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
	+	Projects and products derived from this software may not use the registered trademark "ResourceSpace", nor may "ResourceSpace" appear in their name, without prior written permission from Montala Limited (https://www.montala.com). You may indicate that your project or product incorporates ResourceSpace by using the phrase "Powered by ResourceSpace".

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

So this is definitely still FOSS/BSD-4-Clause. Please revert this commit